doc/admin-guide/plugins/statichit.en.rst

```diff
@@ -0,0 +1,106 @@
+reloadable). It does not use fsnotify for watching the source files.
+
+When the plugin is invoked during remapping, if the transaction already has a status other than OK, the plugin
+will ignore the transaction.
+
+These are the available options for the plugin:
+
+=================  =======  =========  ======  ==============
+Long Name          One      Required?  Has     Description
+                   Letter              Value?
+                   Name
+=================  =======  =========  ======  ==============
+--file-path        -f       Yes        Yes     File path. If the given path is relative, it is relative to the |TS|
+                                               configuration directory.  If the specified file is a directory, it implies
+                                               --disable-exact, and URL path is appended to this directory path to get
+                                               the path for the file containing the body.  If the specified file is not a
+                                               directory, it should contain the body.
+--mime-type        -m       No         Yes     Defaults to "text/plain".
+--max-age          -a       No         Yes     Defaults to "0".  If the value is not 0, it gives the number of seconds the
+                                               static body will remain fresh when cache.  (Including in the proxy's cache, if
+                                               enabled.)
+--failure-code     -c       No         Yes     Defaults to "404" (NOT FOUND).  This response status will be returned if the
+                                               file which should contain the body does not exist.  This option is
+                                               particularly useful when --file-path specifies a directory.
+--success-code     -s       No         Yes     Defaults to "200" (OK). This is the response status when the static object is
+                                               successfully returned in the transaction response.
+--disable-exact    -d       No         No      Disable "exact" URL match.  "Exact" match, for this plugin, means the URL must
+                                               not include a path.
+=================  =======  =========  ======  ==============
+
+When using long option names, the value can be specified by appending = and then the value.  `@pparam=--max-age=5` for
+example.  Otherwise, the value must be specified as the next pparam.  `@pparam=-a @pparam=5` for example.
+
+When exact match is enabled, if the request URL has a path, a response with NOT FOUND status will be sent.
+
+Mime Headers in Responses with OK/200 Status
+============================================
+
+#.  Content-Type.  The value given with the --mime-type parameter, or its default value.
+#.  Content-Length.
+#.  Cache-Control.  If --max-age is 0, the value will be "no-cache".  Otherwise, the value will be "max-age=d", where
```

Comment:
Is it really no-cache or no-store? Probably doesn't end up mattering for this use case, as the origin is still ATS, but no-cache is widely misunderstood, and would be good to not add to confusion :)

Reply:
https://github.com/apache/trafficserver/blob/b46f5d061af455dc75956346da3308788381b30d/plugins/statichit/statichit.cc#L337 .  Core TS seems to add a no-store cache control header for 4xx responses.  (May be affected by negative caching config.)